client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -320,6 +373,7 @@ const PersonalInfoTab: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Element
                                                     methods.setValue(PersonalInfoDataContextFields.INSURANCE_COMPANY, '');
                                                 }
                                             }}
+                                            className={props.value === NO_INSURANCE ? classes.markComplexity : ''}
```

Comment:
cant you use &&?

Reply:
no because it is a boolean indication and not a check that props.value is defined